compiler/src/iree/compiler/Modules/HAL/Loader/Conversion/StreamToHALLoader/test/cmd_ops.mlir

```diff
@@ -89,3 +89,29 @@ func.func @cmdDispatch(%buffer0: !stream.resource<transient>, %buffer0_size: ind
   // CHECK: return %c0
   return %fence : !stream.timepoint
 }
+
+// -----
+
+// CHECK-LABEL: @trace_tensor
+// CHECK-SAME: %[[ARG0:.+]]: !hal.buffer
+func.func @trace_tensor(%arg0: !stream.resource<external>) -> !hal.buffer_view {
+  // CHECK: %[[C180:.+]] = arith.constant 180 : index
+  %c180 = arith.constant 180 : index
+
+  // CHECK: %[[C1:.+]] = arith.constant 1 : index
+  // CHECK: %[[C45:.+]] = arith.constant 45 : index
+  // CHECK: %[[C0:.+]] = arith.constant 0 : index
+  // CHECK: %[[C268435488:.+]] = arith.constant 268435488 : i32
+  // CHECK: %[[C1_i32:.+]] = arith.constant 1 : i32
+  // CHECK: %[[VIEW:.+]] = hal_inline.buffer_view.create buffer(%arg0 : !hal.buffer)[%[[C0]], %[[C180]]] shape([%[[C1]], %[[C45]]]) type(%[[C268435488]]) encoding(%[[C1_i32]]) : !hal.buffer_view
+  %tensor = stream.tensor.export %arg0 : tensor<1x45xi32> in !stream.resource<external>{%c180} -> tensor<1x45xi32>
+
+  // CHECK: hal_inline.buffer_view.trace %[[VIEW]] : !hal.buffer_view attributes {key = "whatevs"}
+  stream.tensor.trace {key = "whatevs"} %tensor : tensor<1x45xi32>
+
+  // CHECK: %[[VIEW:.+]] = hal_inline.buffer_view.create buffer(%[[ARG0]] : !hal.buffer)[%[[C0]], %[[C180]]] shape([%[[C1]], %[[C45]]]) type(%[[C268435488]]) encoding(%[[C1_i32]]) : !hal.buffer_view
+  %buffer = stream.tensor.export %arg0 : tensor<1x45xi32> in !stream.resource<external>{%c180} -> !hal.buffer_view
+
+  // CHECK: return %[[VIEW]] : !hal.buffer_view
+  return %buffer : !hal.buffer_view
```

Comment:
you can remove this extra IR as it's not adding additional testing vs the first one of the same export
```suggestion
```